docs/user/dashboard/lens-advanced.asciidoc

```diff
@@ -57,13 +57,15 @@ image::images/lens_clickAndDragZoom_7.16.gif[Cursor clicking and dragging across
 
 . In the layer pane, click *Count of Records*.
```

Comment:
For another PR, but If this is a label the UI, shouldn't it be sentence capped: Count of records.

Reply:
`Records` is the field name, so `Count of Records` is correct here.